ansible/templates/minigraph_dpg.j2

```diff
@@ -108,7 +108,7 @@
 {% if 'port-channel' in vm_topo_config['vm'][vms[index]]['ip_intf'][dut_index|int]|lower %}
 {% set port_channel_intf=';'.join(intf_names[vms[index]])  %}
         <PortChannel>
-          <Name>PortChannel{{ ((index+1)|string).zfill(4) }}</Name>
+          <Name>PortChannel{{ '10' + ((index+1)|string) }}</Name>
```

Comment:
This diff is merged with https://github.com/Azure/sonic-mgmt/pull/4763

Reply:
yes that should be fine.. git is not showing conflict.